Changes

```diff
@@ -553,6 +553,9 @@ OCaml 5.0
 - #8878: Add String.hash and String.seeded_hash.
   (Tom Kelly, review by Alain Frisch and Nicolás Ojeda Bär)
 
+* #10967: Add Filename.temp_dir (David Turner)
```

Comment:
This is in the 5.0 section, but 5.0 is already released. You should put this in the "Working version" section (you may need to create a "standard library" subsection if if doesn't exist yet).